internal/hcsoci/resources.go

```diff
@@ -7,6 +7,14 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	// supported internal mount types
+	Bind                  string = "bind"
+	PhysicalDisk          string = "physical-disk"
+	VirtualDisk           string = "virtual-disk"
+	ExtensibleVirtualDisk string = "extensible-virtual-disk"
```

Comment:
nit: you don't need the `string` type. that way, the constants can be used with any string-like type

```suggestion
	Bind                  = "bind"
	PhysicalDisk          = "physical-disk"
	VirtualDisk           = "virtual-disk"
	ExtensibleVirtualDisk = "extensible-virtual-disk"
```

Reply:
Added a MountType for string so that it can be used to indicate type of mount instead of string in the PR for adding mounts to running containers too. Is this ok?